model_helen_comments.py

```diff
@@ -0,0 +1,127 @@
+"""Models and database functions for showme db."""
+
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+##############################################################################
+
+class Show(db.Model):
+    """Runway show details."""
+
+    __tablename__ = "shows"
+
+    show_id = db.Column(db.Integer,
+                        autoincrement=True,
+                        primary_key=True)
+    season = db.Column(db.String(6))
+    year = db.Column(db.Integer)
+    brand_id = db.Column(db.Integer,
+                         db.ForeignKey('brands.brand_id'))
+
+    # Define relationship to brands
+    brands = db.relationship('Brand')
+
+    def __repr__(self):
+        return "<Show show_id=%s season=%s brand_id=%s year=%s>" % (
+            self.show_id,
+            self.season,
+            self.brand_id,
+            self.year)
+
+
+class Show_Color(db.Model):
+    """Association table connecting specific color info to shows."""
+
+    __tablename__ = "show_colors"
+
+    show_colors_id = db.Column(db.Integer,
+                               autoincrement=True,
+                               primary_key=True)
+    show_id = db.Column(db.Integer,
+                        db.ForeignKey('shows.show_id'))
+    color_id = db.Column(db.Integer,
+                         db.ForeignKey('colors.color_id'))
+
+    # Define relationship to Show & Color tables
+    shows = db.relationship('Show')
+    colors = db.relationship('Color')
+
+    def __repr__(self):
+        return "<Show_Color show_colors_id=%s show_id=%s color_id=%s" % (
+            self.show_colors_id,
+            self.show_id,
+            self.color_id)
+
+
+class Color(db.Model):
+    """Show specific color name/hex info."""
+
+    __tablename__ = "colors"
+
+    color_id = db.Column(db.Integer,
+                         autoincrement=True,
+                         primary_key=True)
+    color_name = db.Column(db.Unicode)
+    color_hex = db.Column(db.Unicode)
+
+    def __repr__(self):
+        return "<Color color_id=%s color_name=%s>" % (
+            self.color_id,
+            self.color_name)
+
+
+class Brand(db.Model):
+    """Show brand info."""
+
+    __tablename__ = "brands"
+
+    brand_id = db.Column(db.Integer,
+                         autoincrement=True,
+                         primary_key=True)
+    brand_name = db.Column(db.String(50))
```

Comment:
I would suggest a power of 2 as a string size (maybe 64), because sql is likely going to allocate space that is a power of 2 anyway, so you might as well be able to use it!